web-app/src/components/SupportRequests/SupportRequests.js

```diff
@@ -7,25 +7,51 @@ import api from '../../api';
 const statusFriendlyTexts = {
     'requestNoSent': 'Pendiente',
     'requestSent': 'Solicitado',
+    'requestCanceled': 'Cancelado',
     'requestFinished': 'Finalizado',
 };
 
-export const SupportRequestsPure = ({ loading, supportRequests, onRequestSupport, submitting }) => {
+export const SupportRequest = ({request, onCancel}) => {
+    const [canceling, setCanceling] = useState(false);
+    const isCancelable = request.status === "requestNoSent" || request.status === "requestSent";
+    const cancel = () => {
+        setCanceling(true);
+        onCancel().then(() => setCanceling(false));
+    }
+
+    return (
+        <div className="support-request-row">
+            <span>Solicitado el {moment(request.timestamp).format('LLL')} por {request.user.username}. Estado: {statusFriendlyTexts[request.status]}</span>
```

Comment:
In English please, it is easier to translate into other languages later.

Reply:
Actually all the web-app interface is in spanish. I agree that it will be easier from english. But we may prefer to do that job in bulk later, don't we?